include/EVT/dev/LCD.hpp

```diff
@@ -190,6 +187,9 @@ class LCD {
         "Not Set",
         "Not Set",
     };
+
+    /** How many pages each section takes up */
```

Comment:
Also, just generally, why did you choose to change this from a variable to a method? You could just make a variable that gets set in the constructor because it shouldn't ever change